assets/assets.go

```diff
@@ -29,6 +29,13 @@ var initDocPaths = []string{
 	filepath.Join("init-doc", "ping"),
 }
 
+// Asset loads and returns the asset for the given name.
+// It returns an error if the asset could not be found or
+// could not be loaded.
+func Asset(f string) ([]byte, error) {
+	return dir.ReadFile(f)
+}
```

Comment:
This just keeps the old API in place that was generated from `go-bindata`

Reply:
Any reason not to remove this ?

Is this code used by anything else than `go-ipfs` ?

Reply:
This just keeps the old API in place that was being used in the rest of the code. It makes changes be baby step and avoids changing many files at once since that is the exposed API that was given by bindata beforehand.

Reply:
> keeps the old API in place that was being used in the rest of the code

Yes, but can't you just change it in the rest of the code also ? Any reason not to do that ?

> It makes changes be baby step and avoids changing many files at once

I think that make sense but, as multiple commits in a single PR, I don't see any reason to keep this API past this PR.

Code kept for *history* reasons is also code that is confusing. a new comer shouldn't have to use git blame to understand why an API is the way it is. So when breaking is not an issue, I don't see any reason not to do so.

Reply:
I think that clearer.